frontends/web/src/routes/settings/settings.tsx

```diff
@@ -105,6 +105,15 @@ class Settings extends Component<Props, State> {
             });
     }
 
+    private handleToggleExpertFee = (event: Event) => {
+        const target = (event.target as HTMLInputElement);
+        setConfig({
+            backend: {
+                [target.id]: target.checked,
```

Comment:
Seems like this is generic enough that the other `handleToggleCoinControl` and `handleToggleEthereum` can use it.

Reply:
@thisconnect ^^

Reply:
one reinitializes accounts. the other doesn't. Not sure if refactoring this is worth it, but I defer ;)